source/features/hide-markdown-diff.tsx

```diff
@@ -1,11 +1,26 @@
 import './hide-markdown-diff.css';
 import React from 'dom-chef';
+import select from 'select-dom';
 import delegate from 'delegate-it';
 import elementReady from 'element-ready';
 import * as pageDetect from 'github-url-detection';
 
 import features from '.';
 
+// Fix duplicated content in changed quote blocks #4035
+function fixDuplicatedContent(): void {
+	if (!select.exists('.js-code-editor.show-preview')) {
+		return;
+	}
+
+	for (const possibleDuplicate of select.all('.markdown-body .changed > .changed_tag[data-before-tag="blockquote"] + p')) {
+		// Content is duplicated if already present in a sibling <ins> element
+		if (possibleDuplicate.textContent && select('ins', possibleDuplicate.parentElement!)?.textContent?.includes(possibleDuplicate.textContent)) {
+			possibleDuplicate.classList.add('removed');
+		}
+	}
```

Comment:
This works fine, but if it's too much complexity we can drop it and show a warning if `'.markdown-body .changed > .changed_tag[data-before-tag="blockquote"]` matches something. 

Although that doesn't necessarily means there's duplicated content, so it'll cause false warnings. Afaik there's no way in this case to detect duplication without actually checking the content of the elements.

Reply:
I had in mind something simpler like checking if there’s a yellow diff. I’d rather show the message often/always instead of maintaining this code

Reply:
> I had in mind something simpler like checking if there’s a yellow diff

A yellow diff just means that a block was modified. It could be anything (paragraph, header, image, etc.) but afaik the duplicated content bug only occurs with quote blocks (and only if they're completely removed and turned into something else).

> I’d rather show the message often/always instead of maintaining this code

:+1: :ok:

Reply:
Let's leave it there for now, I'd like to see how it works first. Refer to my last comment (I can't replicate the original issue with your demo markdown)

Reply:
I think I found a pure CSS solution to this \o/
Needs more testing though